src/main/java/com/theironyard/invitator/Event.java

```diff
@@ -0,0 +1,158 @@
+package com.theironyard.invitator;
+
+import java.sql.*;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by melmo on 12/22/16.
+ */
+public class Event {
+    private Integer eventId;
+    private Integer userId;
+    private String name;
+    private String date;
+    private String time;
+    private String location;
+    private String description;
+    private Map<Integer, Person> invitedList = new HashMap<>();
+    private Map<Integer, Person> notInvitedList = new HashMap<>();
+
+    public Event(Integer eventId, Integer userId, String name, String date, String time, String location, String description) {
+        this.eventId = eventId;
+        this.userId = userId;
+        this.name = name;
+        this.date = date;
+        this.time = time;
+        this.location = location;
+        this.description = description;
+    }
+
+    public Event(String name, String date, String time, String location, String description, Integer userId){
+        this.eventId = 0;
+        this.userId = userId;
+        this.name = name;
+        this.date = date;
+        this.time = time;
+        this.location = location;
+        this.description = description;
+    }
+
+    public void saveEvent() throws SQLException {
+        Connection connection = DriverManager.getConnection("jdbc:postgresql:invitation");
```

Comment:
As discussed the connection will be created every time someone runs this method or any method with a connection in it.  The better way would be to either pass in a connection that is created in the `main` method or make the connection `static`